app/src/main/java/net/gsantner/markor/activity/DocumentShareIntoFragment.java

```diff
@@ -384,6 +384,7 @@ private void addDocumentToPrefgroup(String filepath, final PreferenceGroup prefG
      * @param link Link url
      * @return formatted URL of format [text](url)
      */
+    // TODO: parameter for AsciiDoc or separate function for AsciiDoc
```

Comment:
for now please lets keep it here as it is, with just reshaping it to markdown-ish syntax.   
Maybe making it format-specific, or settings whatever, is something for later when it's really needed.